frontend/src/pages/home/home.tsx

```diff
@@ -284,12 +284,11 @@ export default class HomePage extends React.Component<HomePageProps, HomePageSta
                     </div>
                     <AnchorSupportedSection anchor={Anchors[0]} onVisible={this.onAnchorVisible}>
                         <div style={{ height: "5rem" }} />
+                        <ImageCard
+                            imageId="cocodiploma"
+                            modalCaption=""
+                            thumbnail="https://cdn.discordapp.com/attachments/745465408848527360/858470646824566814/diploma_final.png" />
```

Comment:
aww, no fade in? That's fine :P

Reply:
I haven't looked closely, but the header also doesn't have a fade in, right? I figure it can just show up the same time as the header.